lib/wsdl.js

```diff
@@ -1112,6 +1112,12 @@ WSDL.prototype._initializeOptions = function (options) {
   } else {
     this.options.returnFault = false;
   }
+  if (options.returnNilAsNull !== undefined) {
```

Comment:
we could set it like `this.options.returnNilAsNull = !!options.returnNilAsNull` which would set it to the expected boolean value in every case.

in general - and especially when looking through your tests - wouldn't it be a better fit if we name the option `handleNilAsNull` instead?